docs/releases/overview.md

```diff
@@ -19,6 +19,67 @@ See the full release notes on [GitHub ↗](https://github.com/vegaprotocol/vega/
 ## Vega core software
 The Vega core software is public on a business-source licence, so you can both view the repository change logs, and refer here for summary release notes for each version that the validators use to run the Vega mainnet. Releases are listed with their semantic version number and the date the release was made available to mainnet validators.
 
+## Pre-release version 0.74 | 2024-01-24
+This version was released to the Vega testnet on 24 January 2024.
+
+This pre-release contains several new features for the Palazzo milestone, including isolated margin, batch proposals, Ethereum RPC and EVM based data sources and a new mark price and price for perps funding TWAP methodology.
+
+### Breaking changes
+
+- Listing transactions on block explorer no longer supports the field `limit`. This has been added in the following [issue ↗](https://github.com/vegaprotocol/vega/issues/10215)
+- Getting a transfer by ID now returns a `TransferNode`. This has been added in the following [issue ↗](https://github.com/vegaprotocol/vega/issues/8056)
+
+### Deprecations
+
+ - Commands `tm` and `tendermint` are deprecated in favour of `cometbft`. This has been added in the following [issue ↗](https://github.com/vegaprotocol/vega/issues/10000)
+
+### New Features
+
+#### Isolated margin
+
+The protocol can now operate in one of two margining modes for each position. The current mode will be stored alongside the party's position record.
+
+* Cross-margin mode (default): this is the mode used by all newly created positions. When in cross-margin mode, margin is dynamically acquired and released as a position is marked to market, allowing profitable positions to offset losing positions for higher capital efficiency (especially with e.g. pairs trades).
+
```

Comment:
```suggestion
* Cross-margin mode (default): this is the mode used by all newly created orders, but it can be changed. When in cross-margin mode, margin is dynamically acquired and released as a position is marked to market, allowing profitable positions to offset losing positions for higher capital efficiency.
```